test/stacktraces.jl

```diff
@@ -0,0 +1,80 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Tests for /base/stacktraces.jl
+
+# Some tests don't currently work for Appveyor 32-bit Windows
+const APPVEYOR_WIN32 = (
+    OS_NAME == :Windows && WORD_SIZE == 32 && get(ENV, "APPVEYOR", "False") == "True"
```

Comment:
d'oh... it may have fixed win32, but locally win64 still had a failure in `make testall1`
```
     * stacktraces          Test Failed
  Expression: catch_backtrace() == StackFrame[]
   Evaluated: [Ptr{Void} @0x000000006d0b869e,Ptr{Void} @0x000000006d0b9680,Ptr{Void} @0x00000002a30eec45,Ptr{Void} @0x00000002a30eed24,Ptr{Void} @0x000000006d0c7eec,Ptr{Void} @0x000000006d09f34a,Ptr{Void} @0x000000006d0c885d,Ptr{Void} @0x000000001b43b00f,Ptr{Void} @0x000000001b43b037,Ptr{Void} @0x000000006d093697,Ptr{Void} @0x000000001b41f749,Ptr{Void} @0x000000001b41f9c7,Ptr{Void} @0x000000006d093697,Ptr{Void} @0x0000000020298f88,Ptr{Void} @0x0000000020299972,Ptr{Void} @0x0000000020299a2e,Ptr{Void} @0x000000006d093697,Ptr{Void} @0x00000000202720b9,Ptr{Void} @0x000000006d093697,Ptr{Void} @0x000000006d0a6d1a,Ptr{Void} @0x000000002026f0be,Ptr{Void} @0x000000006d093697,Ptr{Void} @0x0000000020270cdc,Ptr{Void} @0x0000000020271944,Ptr{Void} @0x000000006d09374f,Ptr{Void} @0x000000006d0b9891] == StackFrame[]
ERROR: LoadError: LoadError: There was an error during testing
 in record(Base.Test.#record, Base.Test.FallbackTestSet, Base.Test.Fail) at .\test.jl:317
 in do_test(Base.Test.#do_test, Base.Test.Returned, Expr) at .\test.jl:215
 in include(Core.#include, UTF8String) at .\boot.jl:264
 in include_from_node1(Base.#include_from_node1, ASCIIString) at .\loading.jl:417
 [inlined code] from .\util.jl:179
 in runtests(#runtests, ASCIIString) at C:\cygwin64\home\Tony\julia\test\testdefs.jl:7
 in run_work_thunk(Base.#run_work_thunk, Base.##247#248{##16#24,Tuple{ASCIIString}}, Bool) at .\multi.jl:714
 in remotecall_fetch(Base.#remotecall_fetch, Any, LocalProcess, ASCIIString, Vararg{ASCIIString}) at .\multi.jl:787
 in remotecall_fetch(Base.#remotecall_fetch, Any, Int64, ASCIIString, Vararg{ASCIIString}) at .\multi.jl:803
 [inlined code] from C:\cygwin64\home\Tony\julia\test\runtests.jl:36
 in #15(##15#23{Array{ASCIIString,1}}) at .\task.jl:431
while loading C:\cygwin64\home\Tony\julia\test\stacktraces.jl, in expression starting on line 45
 in remotecall_fetch(Base.#remotecall_fetch, Any, LocalProcess, ASCIIString, Vararg{ASCIIString}) at .\multi.jl:788
 in remotecall_fetch(Base.#remotecall_fetch, Any, Int64, ASCIIString, Vararg{ASCIIString}) at .\multi.jl:803
 [inlined code] from C:\cygwin64\home\Tony\julia\test\runtests.jl:36
 in #15(##15#23{Array{ASCIIString,1}}) at .\task.jl:431
 in sync_end(Base.#sync_end) at .\task.jl:397
 [inlined code] from .\task.jl:406
 in #11(##11#19) at C:\cygwin64\home\Tony\julia\test\runtests.jl:31
 in cd(Base.Filesystem.#cd, ##11#19, ASCIIString) at .\file.jl:57
 in include(Core.#include, ASCIIString) at .\boot.jl:264
 in include_from_node1(Base.#include_from_node1, UTF8String) at .\loading.jl:417
 in process_options(Base.#process_options, Base.JLOptions) at .\client.jl:244
 in _start(Base.#_start) at .\client.jl:300
while loading C:\cygwin64\home\Tony\julia\test\runtests.jl, in expression starting on line 13
```